lib/api/attributes.js

```diff
@@ -424,7 +419,7 @@ exports.removeAttr = function (name) {
  * @see {@link http://api.jquery.com/hasClass/}
  */
 exports.hasClass = function (className) {
-  return _.some(this, function (elem) {
+  return Array.from(this).some(function (elem) {
```

Comment:
Can we use `toArray` instead of `Array.from`? (Same in other spots.)

```suggestion
  return this.toArray().some(function (elem) {
```